src/task/tests/metrics/inputs.spec.ts

```diff
@@ -38,13 +40,16 @@ describe('inputs.ts', (): void => {
     when(runnerInvoker.getInput(deepEqual(['Always', 'Close', 'Comment']))).thenReturn('')
     when(runnerInvoker.getInput(deepEqual(['File', 'Matching', 'Patterns']))).thenReturn('')
     when(runnerInvoker.getInput(deepEqual(['Code', 'File', 'Extensions']))).thenReturn('')
+    when(runnerInvoker.getInput(deepEqual(['Change', 'Title']))).thenReturn('')
```

Comment:
Move this above `when(runnerInvoker.getInput(deepEqual(['Always', 'Close', 'Comment']))).thenReturn('')` for consistency.